crates/oracle/src/contracts.rs

```diff
@@ -49,7 +49,7 @@ where
             .query("currentEpoch", (), None, Default::default(), None)
             .await?;
         let current_epoch = epoch_number.as_u64();
-        debug!(current_epoch, "Got current epoch from the Epoch Manager");
+        debug!("Epoch Manager is at epoch {current_epoch}");
```

Comment:
This is just for using the same log format as the [Subgraph epoch check](https://github.com/graphprotocol/block-oracle/pull/160/files#diff-e8e985bdb5ff7e8247748de894aaebb17322ce87813dadce3a8f269dcf4dcf7bR146).